app/src/main/res/values/strings.xml

```diff
@@ -159,10 +160,18 @@
     <string name="coins_asset_send">Send %1$s</string>
     <string name="coins_asset_balance">Balance:</string>
     <string name="coins_asset_send_recepient">Recipient\'s address</string>
+    <string name="coins_asset_send_from">From</string>
+    <string name="coins_asset_send_to">To</string>
     <string name="coins_asset_send_amount">Amount</string>
     <string name="coins_asset_send_max">Send max</string>
     <string name="coins_asset_send_review">Review</string>
+    <string name="coins_asset_send_review_hint">You’re about to initiate a Safe transaction. It won’t be submitted to blockchain yet.</string>
```

Comment:
> You’re about to initiate a Safe transaction. It won’t be submitted to blockchain yet.

I would rephrase this to:
You’re about to initiate a Safe transaction. It won’t be submitted to the blockchain yet.
or
You’re about to initiate a Safe transaction. It won’t be submitted to a blockchain yet.

Reply:
But this is not a blocker since I have seen this error in several other places.